lib/jormungandr/src/Cardano/Wallet/Jormungandr/Compatibility.hs

```diff
@@ -228,6 +228,7 @@ genConfigFile stateDir (BaseUrl _ host port path) = object
             [ "messages" .= String "low"
             , "blocks" .= String "normal"
             ]
+        , "public_address" .= String "/ip4/127.0.0.1/tcp/8081"
```

Comment:
What about this?

>What about making the config generator take an extra port param, and find an adequate port in the launch command handler (by querying the system for a random available one)?

I think that putting explicit port may cause some unnecessary troubleshooting on the user side if it interferes with some running service. Especially that the config is automatically generated, and we put this port somewhat silently.

Reply:
@piotr-iohk  @KtorZ  yes, I can add parameter port to `genConfigFile` and add function that uses `bindRandomTcp` which get available port by binding to socket, then closes the socket and then deliver the port that was engaged in it. But how to add this in   `config.yaml` ???

Reply:
`lib/jormungandr/test/data/jormungandr/config.yaml` -> here we can have `8081` I think, it's used for starting `jormungandr` for some portion of integration tests.

Reply:
ok, give me a sec ;-)

Reply:
done